src/app/shared/cookies/browser-klaro.service.ts

```diff
@@ -91,6 +91,10 @@ export class BrowserKlaroService extends KlaroService {
         Klaro.setup(this.klaroConfig);
       });
 
+    if (!environment.info.enablePrivacyStatement) {
+      delete this.klaroConfig.privacyPolicy;
```

Comment:
This works well, but I do notice that there are still links to the privacy policy in Klaro.  In the initial notification modal, it includes the text `To learn more, please read our [privacy policy]`.  That text also exists if you click "Customize" -- it's in the description just under the header.

I only mention this in case there's a way to automatically *remove* this sentence from both places when the privacy policy is disabled.  If it's possible to do automatically, I'd recommend we do it.  If not, then anyone who wants to fully remove the privacy policy will need to modify their i18n (e.g. `en.json5`) to remove the sentence where it exists in those keys.

Reply:
I have implemented a new message in `en.json5` this message does not contain the "please read our" part and will be used when the privacy policy is not enabled.